pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -114,15 +114,13 @@ def qfunc_transformed(*args, **kwargs):
 
     def _qnode_transform(self, qnode, targs, tkwargs):
         """Apply the transform on a QNode. It populates the transform program of a QNode"""
-        if not qnode.transform_program:
+        if qnode.transform_program.is_empty():
             qnode = copy.deepcopy(qnode)
 
         if self.expand_transform:
             qnode.add_transform(TransformContainer(self._expand_transform))
         qnode.add_transform(
-            TransformContainer(
-                self._transform, targs, tkwargs, self._classical_cotransform, self._is_informative
-            )
+            TransformContainer(self._transform, targs, tkwargs, self._classical_cotransform)
```

Comment:
Why has `is_informative` been removed from from the args?

Reply:
Merge conflict, good catch :+1: